Description: ignore_scp_dbus_failures
 add ignore_scp_dbus_failures.diff for LP: #902762
Forwarded: not-needed
Origin: vendor, https://bugs.launchpad.net/ubuntu/+source/system-config-printer/+bug/902762
Author: Jonathon Riddell <jr@kubuntu.org>
====================================================================
Index: print-manager/add-printer/DevicesModel.cpp
===================================================================
--- print-manager.orig/add-printer/DevicesModel.cpp
+++ print-manager/add-printer/DevicesModel.cpp
@@ -339,7 +339,7 @@ void DevicesModel::getGroupedDevicesFail
 {
     qWarning() << error <<  message;
     groupedDevicesFallback();
-    emit errorMessage(i18n("Failed to group devices: '%1'",error.message()));
+    //    emit errorMessage(i18n("Failed to group devices: '%1'",error.message()));
     emit loaded();
 }
 
Index: print-manager/libkcups/SelectMakeModel.cpp
===================================================================
--- print-manager.orig/libkcups/SelectMakeModel.cpp
+++ print-manager/libkcups/SelectMakeModel.cpp
@@ -240,7 +240,7 @@ void SelectMakeModel::getBestDriversFail
     // Show the PPDs anyway
     m_gotBestDrivers = true;
     ui->messageWidget->setText(i18n("Failed to search for a recommended driver: '%1'", error.message()));
-    ui->messageWidget->animatedShow();
+    //ui->messageWidget->animatedShow();
     setModelData();
 }
 
